static/js/browser2/browser.ts

```diff
@@ -19,22 +19,52 @@ import ReactDOM from "react-dom";
 import _ from "lodash";
 
 import { BrowserPage, nodeTypeEnum } from "./browser_page";
+import axios from "axios";
+import { removeLoadingMessage } from "./shared";
 
 window.onload = () => {
   const dcid = document.getElementById("node").dataset.dcid;
   const nodeName = document.getElementById("node").dataset.nn;
   const urlParams = new URLSearchParams(window.location.search);
   const statVarId = urlParams.get("statVar") || "";
-  const nodeType = _.isEmpty(statVarId)
-    ? nodeTypeEnum.GENERAL
-    : nodeTypeEnum.PLACE_STAT_VAR;
-  ReactDOM.render(
-    React.createElement(BrowserPage, {
-      dcid,
-      nodeType,
-      nodeName,
-      statVarId,
-    }),
-    document.getElementById("node")
-  );
+  axios
+    .get(`/api/browser/propvals/typeOf/${dcid}`)
+    .then((resp) => {
+      const values = resp.data.values;
+      const types = values.out ? values.out : [];
+      const listOfTypes = types.map((type) => type.dcid);
+      ReactDOM.render(
+        React.createElement(BrowserPage, {
+          dcid,
+          nodeType: getNodeType(listOfTypes, statVarId),
+          nodeName,
+          statVarId,
+        }),
+        document.getElementById("node")
+      );
+    })
+    .catch(() => removeLoadingMessage());
 };
+
+function getNodeType(listOfTypes, statVarId): string {
+  if (!_.isEmpty(statVarId)) {
+    return nodeTypeEnum.PLACE_STAT_VAR;
+  }
+  let type = "";
+  if (listOfTypes.length > 0) {
+    type = listOfTypes[0];
+  }
+  for (const targetType of ["State", "County", "City"]) {
```

Comment:
could we pull out the list of types to a file constant? that also helps to make it self documenting

Reply:
it's also not immediately clear why state and county are part of "place_stat_var", please add a comment

Reply:
what do you mean state and county are part of place_stat_var? Right now, the nodeType is considered place_stat_var if there is a statvar defined. But I guess place_stat_var is more of a pageType than a nodeType and I should do a better job of distinguishing the two

Reply:
I'm not sure pulling out the list of types to a file constant is necessary. This line was copied from the old kg. I believe this line is there because a node can have multiple typeOf values, but if the list contains one of those 3 types, then we wanted to use that to decide what is rendered for the node.

Reply:
I agree here is more about the "page type"